src/server.js

```diff
@@ -5,6 +5,19 @@ require("dotenv").config();
 const { app } = require("./utilities/bolt.js");
 const { processMessage } = require("./utilities/helperFunctions.js");
 
-app.message(async ({ message }) => {
-  await processMessage(message);
+//globals
+
+//for dev
+let DEV;
```

Comment:
How about just setting dev = to this

Reply:
There are a few ways to skin this cat. If you do a `const DEV = ` whatever in the if statement, it's scoped and can't be accessed in the rest of the code. I could do a ternary or something 🤷‍♂️